src/loaders/FileLoader.js

```diff
@@ -239,7 +226,155 @@ class FileLoader extends Loader {
 
 		scope.manager.itemStart( url );
 
-		return;
+		return res;
```

Comment:
See [this comment](https://github.com/mrdoob/three.js/pull/22510#discussion_r714249173). I think it's best if we don't return anything from `onLoad` unless we have an explicit understanding of the use case.

Reply:
oh, I put that one back because it was [in the original](https://github.com/mrdoob/three.js/blob/8cb903d61618cb58e3134431a389212d9c98dff0/src/loaders/FileLoader.js#L276). I mean, I suppose not returning anything shouldn't change anything, right?